src/main/java/org/example/controller/MainController.java

```diff
@@ -0,0 +1,45 @@
+package org.example.controller;
+
+import org.example.MainServlet;
+import org.example.model.Arguments;
+import org.example.model.Response;
+import org.example.service.HelloService;
+import org.example.service.NotFoundService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MainController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(MainController.class);
+    private final HelloService helloService = HelloService.getInstance();
+    private final NotFoundService notFoundService = NotFoundService.getInstance();
+
+    private static MainController instance;
+
+
+    public Response getHelloType(Arguments arguments) {
+        Response response = new Response();
+        response.setContentType("text/html");
```

Comment:
Create separate enum for the context types

Reply:
fixed